### PR TITLE
Some careless review on compile.sh

### PR DESCRIPTION
**Pull request by Arthur2e5**
Just some routine, machine-doable linting done by a bored human.

(Not bored enough to write `set /p` and `IF %BLAH% == BLAH` yet.)

### CODE REVIEW

**Comment by Schwertspize** [comment]
Looks good for me

**Comment by LogicParrot** [comment]
Just putting this here without actually checking the code:
Have you ensured all the changes are POSIX-compliant? We're not just aiming for BASH.

**Comment by Arthur2e5** [comment]
All the changes -- Yes.
All lines in the code -- I may have missed something.

**Comment by LogicParrot** [comment]
The previous code was already POSIX-compliant.

**Comment by Arthur2e5** [comment]
No, parentheses in [ are not POSIX at all.

Neither are -a -o.

**Comment by LogicParrot** [comment]
My apologies.

File: compile.sh [hunk 1]
```diff
@@ -93,23 +92,23 @@ if [ "$MISSING_PACKAGES" != "" ]; then
 
 	# apt-get guide.
 	apt-get --help > /dev/null 2> /dev/null && \
-	missingDepsExit "sudo apt-get install$MISSING_PACKAGES"
+	missingDepsExit "sudo apt-get install $MISSING_PACKAGES"
```
Comment (LogicParrot) [inline compile.sh]: These were stuck together for a reason. It's because the printf above prints `printf %s "<SPACE>$PACKAGE_NAME"`

Comment (LogicParrot) [inline compile.sh]: In other words, now this is what's printed when e.g. make is missing:
`sudo apt-get install<space><space>make`

Comment (Arthur2e5) [inline compile.sh]: Hmm… My fault.

Comment (LogicParrot) [inline compile.sh]: No problem. That's very minor. I'll fix it. Thanks for the cleanup :)

**Pull request merged**